src/strongbox/core.clj

```diff
@@ -646,10 +647,23 @@
   "merges the data from an installed addon with it's match in the catalogue"
   [installed-addon :addon/toc, db-catalogue-addon :addon/summary]
   (let [;; nil fields are removed from the catalogue item because they might override good values in the .toc or .nfo
-        db-catalogue-addon (utils/drop-nils db-catalogue-addon [:description])]
+        db-catalogue-addon (utils/drop-nils db-catalogue-addon [:description])
```

Comment:
add a todo to revisit this, I can't see any null/nil values in the catalogue anymore.